bash_completions.go

```diff
@@ -216,7 +220,13 @@ func postscript(out *bytes.Buffer, name string) {
 }
 
 `, name)
-	fmt.Fprintf(out, "complete -F __start_%s %s\n", name, name)
+	fmt.Fprintf(out, `
+		if builtin compopt > /dev/null 2>&1; then
+			complete -F __start_%s %s
+		else
+			complete -o nospace -F __start_%s %s
```

Comment:
This means that on OS X things will just never have a space after the tab'd item...   is that preferable?